

echo "# Start Merge"

python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1_0
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1_1
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1_2
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1_4
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1.5_0
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1.5_1
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1.5_2
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1.5_4
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1.5_8
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_1_8
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2_0
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2_1
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2_2
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2_4
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2.5_0
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2.5_1
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2.5_2
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2.5_4
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2.5_8
python3 merge_generated_problems.py --output_format original generated_problems/analysis/output_original_caption_axiom_remapping/ generated_problems/analysis/output_original_sine_2_8

echo "# FINISHED #"
